.gitignore

```
@@ -1 +1,7 @@
-.idea/
\ No newline at end of file
+.idea/
+__pycache__/
```

Can you also add the below:
```suggestion
__pycache__/
openstack-rabbit-consumer/rabbit_consumer/__pycache__/
openstack-rabbit-consumer/tests/__pycache__/
```
Or remove the files from the branch